exporters/zipkin/src/zipkin_exporter.cc

```diff
@@ -105,7 +106,8 @@ sdk::common::ExportResult ZipkinExporter::Export(
     }
     return sdk::common::ExportResult::kFailure;
   }
-  return sdk::common::ExportResult::kSuccess;
+  // Commenting this return statement as the code is unreachable.
+  // return sdk::common::ExportResult::kSuccess;
```

Comment:
Good find, as both branches in the previous if/else have a return.

Please remove the line then instead of commenting it.

Reply:
Removed the comment and return statement.